Plugins/Geant4/include/Acts/Plugins/Geant4/Geant4Converters.hpp

```diff
@@ -16,14 +16,16 @@
 #include <memory>
 #include <tuple>
 
+
 namespace CLHEP {
 class Hep3Vector;
 class HepRotation;
 }  // namespace CLHEP
 
-namespace HepGeom {
-class Transform3D;
-}
+#include "/home/cms/software/geant4/install_dir/include/Geant4/G4RotationMatrix.hh"
```

Comment:
These include files need to be directly from Geant4 if needed, though I guess you don't need them in the header, right?